lib/datadog/security/reactive/subscriber.rb

```diff
@@ -0,0 +1,15 @@
+module Datadog
+  module Security
+    module Reactive
+      class Subscriber
+        def initialize(&block)
```

Comment:
Since this class is effectively a wrapper for the block/proc received, would it make sense to just eliminate it and keep a list of procs in the engine instead?

Reply:
This kind of wrapping helps a lot with typing. Also in a more involved implementation there were more methods and attributes in there, this was trimmed down as it is now but I anticipate it to evolve back.